### PR TITLE
add release 1-9 ci/periodic bazel jobs

### PR DESCRIPTION
**Pull request by BenTheElder**
/area jobs

### CODE REVIEW

**Comment by BenTheElder** [comment]
xref: https://github.com/kubernetes/test-infra/issues/5558

**Comment by krzyzacy** [comment]
/cc @luxas 
for kubeadm

File: jobs/config.json [hunk 1]
```diff
@@ -8090,6 +8106,44 @@
...
+      "--provider=kubernetes-anywhere",
+      "--test_args=--ginkgo.focus=\\[Conformance\\] --minStartupPods=8",
+      "--timeout=300m"
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "sig-cluster-lifecycle"
+    ]
+  },
+  "ci-kubernetes-e2e-kubeadm-gce-1-9": {
```
Comment (luxas) [inline jobs/config.json]: As we discussed earlier, we might ditch this job as it is exactly the same as `periodic-kubernetes-e2e-kubeadm-gce-1-9` but postsubmit. On release branches periodic jobs tend to be easier to get a signal from. WDYT?

File: jobs/config.json [hunk 2]
```diff
@@ -8090,6 +8106,44 @@
...
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=ci/latest-1.8",
+      "--gcp-zone=us-central1-f",
+      "--kubeadm=ci",
+      "--kubernetes-anywhere-kubelet-ci-version=latest-1.8",
+      "--kubernetes-anywhere-kubernetes-version=latest-1.8",
+      "--provider=kubernetes-anywhere",
+      "--test_args=--ginkgo.focus=\\[Conformance\\] --minStartupPods=8",
```
Comment (luxas) [inline jobs/config.json]: this should have the full `\\[Conformance\\]|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\]` coverage; as this tests a v1.8 API server which supports this thing (compared to testing v1.7 earlier in the similar job)

File: jobs/config.json [hunk 3]
```diff
@@ -9819,6 +9908,29 @@
...
+  "periodic-kubernetes-e2e-kubeadm-gce-upgrade-1-8-1-9": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=release/latest-1.9",
+      "--extract=ci/latest-1.9",
+      "--gcp-zone=us-central1-f",
+      "--kubeadm=periodic",
+      "--kubernetes-anywhere-kubelet-ci-version=latest-1.9",
```
Comment (luxas) [inline jobs/config.json]: seems like this should be 1.8

File: jobs/config.json [hunk 4]
```diff
@@ -9819,6 +9908,29 @@
...
+      "--kubernetes-anywhere-kubernetes-version=latest-1.9",
```
Comment (luxas) [inline jobs/config.json]: seems like this should be 1.8

File: jobs/config.json [hunk 5]
```diff
@@ -9819,6 +9908,29 @@
...
+      "--kubernetes-anywhere-upgrade-method=upgrade",
+      "--provider=kubernetes-anywhere",
+      "--skew",
+      "--test_args=--ginkgo.focus=\\[Conformance\\]|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\] --minStartupPods=8",
+      "--timeout=300m",
+      "--upgrade_args=--ginkgo.focus=\\[Feature:MasterUpgrade\\] --upgrade-target=release/latest-1.8"
```
Comment (luxas) [inline jobs/config.json]: seems like this should be 1.9 I think

File: jobs/config.json [hunk 6]
```diff
@@ -9819,6 +9908,29 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "periodic-kubernetes-e2e-kubeadm-gce-upgrade-1-8-1-9": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=release/latest-1.9",
```
Comment (luxas) [inline jobs/config.json]: "--extract=release/latest-1.9",
      "--extract=ci/latest-1.8",

File: testgrid/config/config.yaml [hunk 7]
```diff
@@ -2642,6 +2676,18 @@ dashboards:
...
+    test_group_name: periodic-kubernetes-bazel-build-1-9
+  - name: bazel-test-1.9
+    test_group_name: periodic-kubernetes-bazel-test-1-9
+  - name: gce-kubeadm-1.9
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-9
+  - name: gce-kubeadm-1.8-on-1.9
+    test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-8-on-1-9
+  - name: periodic-gce-kubeadm-1.9
+    test_group_name: periodic-kubernetes-e2e-kubeadm-gce-1-9
+  - name: gce-kubeadm-upgrade-1.8-1.9
```
Comment (luxas) [inline testgrid/config/config.yaml]: Add these kubeadm jobs to `sig-cluster-lifecycle-all` as well?

**Comment by BenTheElder** [comment]
/hold 
Will fix comments in the morning, thanks for reviewing @luxas :-)

File: jobs/config.json (hunk 1 above)
Comment (BenTheElder) [reply jobs/config.json]: SGTM, I'll remove this.

File: jobs/config.json (hunk 5 above)
Comment (BenTheElder) [reply jobs/config.json]: done. too many versions to bump :joy:

File: jobs/config.json (hunk 4 above)
Comment (BenTheElder) [reply jobs/config.json]: done.

File: jobs/config.json (hunk 3 above)
Comment (BenTheElder) [reply jobs/config.json]: done.

File: jobs/config.json (hunk 6 above)
Comment (BenTheElder) [reply jobs/config.json]: done.

File: jobs/config.json (hunk 2 above)
Comment (BenTheElder) [reply jobs/config.json]: done.

File: testgrid/config/config.yaml (hunk 7 above)
Comment (BenTheElder) [reply testgrid/config/config.yaml]: I think they're all in this dashboard now.

**Comment by BenTheElder** [comment]
/hold cancel
I think all comments should be addressed now.

File: prow/config.yaml [hunk 8]
```diff
@@ -19955,6 +20085,181 @@ periodics:
...
+      - name: cache-ssd
+        hostPath:
+          path: /mnt/disks/ssd0
+  - name: periodic-kubernetes-e2e-kubeadm-gce-1-9
+    agent: kubernetes
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/e2e-kubeadm:v20171108-e53c04a8
+        args:
+        - "--repo=k8s.io/kubernetes=release-1.8"
```
Comment (luxas) [inline prow/config.yaml]: there is a typo here, should be v1.9 ref https://github.com/kubernetes/test-infra/issues/5769
cc @enisoc @BenTheElder ^

**Pull request merged**